dashboard/data/file_preprocessing/echo_files_parser.py

```diff
@@ -98,6 +115,12 @@ def retain_key_columns(self, columns: list[str] = None) -> EchoFilesParser:
 
         retain_echo = [col for col in columns if col in self.echo_df.columns]
         self.echo_df = self.echo_df[retain_echo]
+        try:
+            self.echo_df["Destination Well"] = self.echo_df[
+                "Destination Well"
+            ].str.replace(r"0(?!$)", "", regex=True)
+        except:
```

Comment:
We should probably do some more concrete error handling (or at least some logging)

+ please define which exception/exceptions should be handled (we should not leave raw `except` as it will catch every single thing that might stop execution including CTRL+C and System Exit - more on that here https://stackoverflow.com/questions/21553327/why-is-except-pass-a-bad-programming-practice)

Reply:
You are right, actually this exception is not necessary, I used it while developing the code.